src/types.ts

```diff
@@ -224,8 +224,101 @@ export interface TransactionArguments {
     treatAsGrossAmount?: boolean;
     forceSweep?: boolean;
     feePayerInfo?: TransactionArgumentsFeePayerInfo;
+    travelRuleMessage?: TravelRule;
 }
 
+type OwnershipProof = {
+    type: string;
+    proof: string;
+};
+
+enum AddressTypeCode {
+    HOME = "HOME",
+    BIZZ = "BIZZ",
+    GEOG = "GEOG"
+}
+
+type Address = {
+    addressType?: AddressTypeCode;
+    department?: string;
+    subDepartment?: string;
+    streetName?: string;
+    buildingNumber?: string;
+    buildingName?: string;
+    floor?: string;
+    postBox?: string;
+    room?: string;
+    postCode?: string;
+    townName?: string;
+    townLocationName?: string;
+    districtName?: string;
+    countrySubDivision?: string;
+    addressLine?: string[];
+    country?: string;
+};
```

Comment:
Why not to export `OwnershipProof`, `AddressTypeCode` & `Address`?
If you do decide to export them then please rename the types to have more context (i.e. "Address" is pretty broad)

Reply:
Export and update naming